templates/tfengine/recipes/resources.hcl

```diff
@@ -272,6 +272,10 @@ schema = {
             description = "Name of the network."
             type        = "string"
           }
+          tier = {
+            description = "The tier for the master instance."
```

Comment:
Can you link to a definition of what "tier" is?

Reply:
Done.